### DISCUSSION TITLE
How does the script load lazy-loaded images without scrolling down the page?

### DISCUSSION THREAD

**Discussion by smithnanic** [gildas-lormeau/SingleFile#1472] [Q&A]
I'm incorporating SingleFile into a Playwright screenshot project, and its working very well so far. The issue is that I have to scroll up and down the page in Playwright in order to trigger all lazy loaded images. 

The SingleFile chrome extension seems to save webpages with the full images (not the initial lazy ones). Is there a specific file or line of code that handles this so I can import it into my Playwright project?

**Comment by gildas-lormeau** [comment]
Is the option `loadDeferredImages` enabled you when run the code of SingleFile? The code that manages the loading of deferred images revolves around this option and is already present, a priori, in your project.
